index.html

```diff
@@ -5,9 +5,12 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width">
 
-    <link href="//fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
+    <link href="http://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
```

Comment:
FYI, This is a protocol relative URL. If the site is http then it would use the http version of the font and if the site is https then it would the https version. It used to be best practice but turns out I was wrong, it has changed, the best practice now is to just use `https` all the time. https://www.paulirish.com/2010/the-protocol-relative-url/

Additionally, [the site](https://cork.js.org/) is on https, so browsers will actually give an insecure content warning  because we have included non-secure assets on the site.